helm/net-exporter/templates/daemonset.yaml

```
@@ -53,16 +53,17 @@ spec:
           {{- if (.Values.NetExporter.DNSCheck.TCP.Disabled) }}
           - "-disable-dns-tcp-check={{ .Values.NetExporter.DNSCheck.TCP.Disabled }}"
           {{- end }}
+          - "-port={{ .Values.port }}"
```

```suggestion
          - "-port={{ .Values.port }}"
        ports:
        - name: http
          port: {{ .Values.port }}
  	  containerPort: {{ .Values.port }}
```

Then the probes can use the named port and we have a visual indication of the port exposed